app/values/pairtree_derivative_path.rb

```diff
@@ -9,17 +9,35 @@ def file_name
   end
 
   def extension
+    pdf_extensions || geo_extensions || other_extensions
```

Comment:
The list of cases is getting pretty big — and it might grow quite a bit more if we add support for A/V or other kinds of files later.  Maybe having a hash mapping destinations to extensions would be simpler?